src/functions/avl-processor/index.ts

```diff
@@ -80,6 +85,7 @@ export const processSqsRecord = async (
     avlSubscriptionTableName: string,
     avlValidationErrorTableName: string,
     gtfsTripMapsTableName: string,
+    enableCancellations: boolean,
```

Comment:
I'd be tempted to just access the env vars in the functions they're being used to save passing them around:

```ts
const enableCancellations = process.env.ENABLE_CANCELLATIONS === "true"
```